source/options.js

```diff
@@ -1,3 +1,4 @@
+import browser from 'webextension-polyfill';
```

Comment:
No longer globally-available; This is required until https://github.com/fregante/browser-extension-template/issues/48 is resolved